doc/lsst.fgcmcal/tasks/lsst.fgcmcal.fgcmCalibrateTract.FgcmCalibrateTractTask.rst

```diff
@@ -0,0 +1,105 @@
+
+######################
+FgcmCalibrateTractTask
+######################
+
+``FgcmCalibrateTractTask`` will run the full ``fgcmcal`` processing cycle on a single tract with multi-band coverage.  Note that the results will not be as robust as a full global calibration because of the limited ability to tract instrumental changes from observations in a single tract.  Running ``FgcmCalibrateTractTask`` requires a look-up table generated by :doc:`lsst.fgcmcal.fgcmMakeLut.FgcmMakeLutTask`.
+
+``FgcmCalibrateTractTask`` is available as a :ref:`command-line task <pipe-tasks-command-line-tasks>`, :command:`fgcmCalibrateTract.py`.
+
+.. _lsst.fgcmcal.fgcmCalibrateTract.FgcmCalibrateTractTask-summary:
+
+Processing summary
+==================
+
+``FgcmCalibrateTractTask`` runs the full ``fgcmcal`` processing, from building star lists to fitting to the output of final products.  Specifically, it will:
+
+#. Build the star lists: :doc:`lsst.fgcmcal.fgcmBuildStars.FgcmBuildStarsTask`
+
+#. Run the fitter until convergence: :doc:`lsst.fgcmcal.fgcmFitCycle.FgcmFitCycleTask`
+
+#. Output the final data products: :doc:`lsst.fgcmcal.fgcmOutputProducts.FgcmOutputProductsTask`
+
+.. _lsst.fgcmcal.fgcmCalibrateTract.FgcmCalibrateTractTask-cli:
+
+fgcmCalibrateTract.py command-line interface
+============================================
+
+.. code-block:: text
+
+   fgcmCalibrateTract.py REPOPATH [@file [@file2 ...]] [--output OUTPUTREPO | --rerun RERUN] [--id] [other options]
+
+Key arguments:
+
+:option:`REPOPATH`
+   The input Butler repository's URI or file path.
+
+Key options:
+
+:option:`--id`:
+   The data IDs to process.  The ``tract`` must be specified.
+
+.. seealso::
+
+   See :ref:`command-line-task-argument-reference` for details and additional options.
+
+.. _lsst.fgcmcal.fgcmCalibrateTract.FgcmCalibrateTractTask-api:
+
+Python API summary
+==================
+
+.. lsst-task-api-summary:: lsst.fgcmcal.fgcmCalibrateTract.FgcmCalibrateTractTask
+
+.. _lsst.fgcmcal.fgcmCalibrateTract.FgcmCalibrateTractTask-butler:
+
+Butler datasets
+===============
+
+When run as the ``fgcmCalibrateTract.py`` command-line task, or directly through the `~lsst.fgcmcal.fgcmCalibrateTract.FgcmCalibrateTractTask.runDataRef` method, ``FgcmCalibrateTractTask`` obtains datasets from the input Butler data repository and persists outputs to the output Butler data repository.
+Note that configurations for ``FgcmCalibrateTractTask``, and its subtasks, affect what datasets are persisted and what their content is.
+
+.. _lsst.fgcmcal.fgcmCalibrateTract.FgcmCalibrateTractTask-butler-inputs:
+
+Input datasets
+--------------
+
+``camera``
+   Camera geometry and detector object
+``fgcmLookupTable``
+    FGCM look-up table produced by :doc:`lsst.fgcmcal.fgcmMakeLut.FgcmMakeLutTask`
+
+.. _lsst.fgcmcal.fgcmCalibrateTract.FgcmCalibrateTractTask-butler-outputs:
+
+Output datasets
+---------------
+
+``ref_cat``
+    Reference catalog of standard stars.
```

Comment:
`ref_cat` is quite a grab bag of data that don't follow the "typical" butler dataset management... It might be helpful to have more descriptions what this `ref_cat` is.